package.json

```diff
@@ -21,60 +21,63 @@
     "lint": "npm i --legacy-peer-deps && npm i --legacy-peer-deps --prefix ./src/ui && eslint --ext .ts src/ui && npm i --legacy-peer-deps --prefix ./src/api && eslint --ext .ts src/api && eslint --ext .js e2e-tests",
     "lint:fix": "prettier --write src/ui  && eslint --ext .ts src/api --fix && eslint --ext .ts src/ui --fix &&  eslint --ext .js e2e-tests --fix",
     "lint:fix:all": "prettier --write src/ui  && eslint --ext .ts src/api --fix && eslint --ext .ts src/ui --fix &&  eslint --ext .js e2e-tests --fix",
+    "prettier": "prettier --no-error-on-unmatched-pattern --log-level error --check **/*.{ts,js}",
```

Comment:
I don't believe we need this, since `prettier` is run alongside the `lint` commands?

Reply:
I have left the `prettier` npm script however temporarly disabled the check in `lint.yml` as I first need to unify `prettier` and `ESLint` configurations.